Shopping-Stories/shopping-stories
# Pull main into preprod

* Testing webhook

* Checking autodeploy

* Verify commit to preprod updates preprod

* Test auto redeploy

* Test redeploy of website again

* Final test

* try 1 more time

* Try agian

* Last test, problem fixed

* Update preprod to current main (#70)

* Ensure website does not redeploy on push to main

* GraphQL defs and basic Graphview (#52)

* Graph view skeleton + some fixes to entry search

* generated types/query-hooks for all queries.

* Updated graphql defs; organization and fragments

* Updated graphql defs; removed "$populate" arguments and directives

* Updated graphql defs; fixed typo on "categoryFields" updated graphq.ts;
regenerated types/hooks for graph ql

* Updated graphql defs: removed "populate" fields for testing, removed "query" suffixes on queries updated graphql.ts: regenerated types/hooks for graphql

* Graph View Page prototype functioning

* Graph View Page Working Build

* Fix openssl-legacy option and compiler warnings

* Remove openssl-legacy option

* Fix jovan messing up the UI

Co-authored-by: Mahlon Scott <1kingjms@gmail.com>

* Restore basic search functionality

* fix null error

* Add debit/credit to display

* Start adding parser output editing page

* Add basic editing functionality to results editor

* Continue work on parser results editor

* Add basic functionality to check progress

* Basic editing and upload done

* fix prod building

Co-authored-by: J-VOL <57736432+J-VOL@users.noreply.github.com>

* fix preprod

* fix merge

* move to https

* fix preprod issues

* fix port for new api endpoints

* test redeploy

Co-authored-by: J-VOL <57736432+J-VOL@users.noreply.github.com>
